File: pset/pset1/mario/mario less/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    // TODO: Prompt for size
    int Height = 0;
    while (Height < 1 || Height > 8)
    {
        Height = get_int("Height: ");
    }


    //loop
    for (int i = 1; i <= Height; i++)
    {
        for (int j = 1; j <= Height - i; j++) //loop for space
        {
            printf(" ");
        }
        for (int k = 1; k <= i; k++) //loop for #
        {
            printf("#"); //
        }
        printf("\n");
    }
}

File: pset/pset6/mario_less/mario.py
from cs50 import get_int

height = get_int("Height: ")
while height < 1 or height > 8:
    height = get_int("Height: ")

for i in range(height):
    for j in range(height):
        if i + j >= height - 1:
            print("#", end="")
        else:
            print(" ", end="")
    print()